src/System.Private.CoreLib/shared/System/String.Comparison.cs

```diff
@@ -755,52 +755,62 @@ public override int GetHashCode()
 
         // Use this if and only if you need the hashcode to not change across app domains (e.g. you have an app domain agile
```

Comment:
I guess this comment should be replaced with a note to only use it if DOS is not a concern or is otherwise mitigated?

Reply:
Updated to:
```
// Use this if and only if 'Denial of Service' attacks are not a concern (i.e. never used for free-form user input),
// or are otherwise mitigated
```